packages/firebase_in_app_messaging/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.0
+
+ - **CHORE**: bump packages.
+ - **BREAKING** **FEAT**: Migrate firebase_in_app_messaging to sound null safety
```

Comment:
Changelogs are generated, could you remove this?

Reply:
done